optaplanner-docs/src/main/asciidoc/Integration/Integration-chapter.adoc

```diff
@@ -481,7 +481,7 @@ To get decent <<logging,logging of the solver(s)>>, create a file `src/main/reso
 ==== Skinny war on WildFly and JBoss EAP
```

Comment:
methodologyOverview.png has too much whitespace at the top

Reply:
it also lost the "swing examples" box. The idea is that the "swing UI code" sits in the optaplanner-examples.jar, but the "examples domain/app code" can live without it.

And while we're working on this, maybe we should add "optaplanner-test", our "unit test" helping jar?

Reply:
The "swing examples" box was not lost, it was removed. There are no other examples than Swing examples now, so no point in having two boxes for that.